src/entities/viewer/index.ts

```diff
@@ -0,0 +1,3 @@
+export * as viewerModel from './model/store';
+export * from './ui/login';
```

Comment:
use index in ./ui folder and import everything from there

Reply:
fixed